script/get_unused_message_id_category.py

```diff
@@ -18,6 +18,8 @@ def register_all_checkers_and_plugins(linter: "PyLinter") -> None:
 
 def get_next_code_category(message_ids: List[str]) -> int:
     categories = sorted({int(i[:2]) for i in message_ids})
+    # 16 was the deleted python 3k+ checker.
+    categories.append(16)
```

Comment:
This should probably be a constant somewhere? Perhaps something like `DEPRECATED_MSGID_PREFIXES` and `DEPRECATED_MSGIDS` in `const.py`?

Reply:
[916e3df](https://github.com/PyCQA/pylint/pull/5839/commits/916e3df4d3c95f76d9e63609df97316c974db2d7)